pxtblocks/blocklyloader.ts

```diff
@@ -3199,4 +3212,18 @@ namespace pxt.blocks {
             }
         }
     }
+
+    let iconCanvasCache: Map<string> = {};
```

Comment:
this name is quite misleading it should be jresIconCache

Reply:
Yeah, legacy name, that I copied over. Will change